.github/workflow_scripts/env_setup.sh

```diff
@@ -53,6 +53,7 @@ function install_multimodal {
     mim install mmcv-full --timeout 60
     python3 -m pip install --upgrade "mmdet>=2.28, <3.0.0"
     python3 -m pip install --upgrade "mmocr<1.0"
+    python3 -m pip install --upgrade git+https://github.com/IDEA-Research/GroundingDINO.git
```

Comment:
Will this install the package from mainline? If so, this could lead to problem if the mainline is not stable (which is going to happen sometime more than likely). @Innixma @gradientsky Any opinions?

Reply:
We only auto install it in CI, and ask users to install it manually when they try to use `open_vocabulary_detection`. I'll fork this repo and install from my github repo to make sure CI is not broken due to any unstable update, and will ask the author when it will be available on PYPI.

Reply:
They just had a Pypi release. I'll test if it's working well and switch to it

Reply:
The only pypi release does not seem like official. Just added an issue asking about official pypi release. But before the official release is available, shall we fork the repo or use the unofficial release?